modules/sdk/src/main/scala/org/tessellation/sdk/cli/CliMethod.scala

```diff
@@ -101,4 +102,6 @@ trait CliMethod {
     PriorityPeerIds.get(environment)
   )
 
+  def makeStateChannelAllowance(metagraphId: Address, peerIdValues: NonEmptySet[String]): Map[Address, NonEmptySet[PeerId]] =
```

Comment:
This method will only be useful for this particular situation we are in now where we only allow one metagraph.